truffle-config.arbitrum.js

```diff
@@ -4,6 +4,12 @@ require('dotenv').config();
 const mnemonic = process.env["MNEMONIC"];
 const infuraKey = process.env["INFURA_KEY"];
 
+//uncomment to use goerliMnemonic, be sure to set it in the .env file
+//const goerliMnemonic = process.env["GOERLI_MNEMONIC"];
+
+//uncomment to use rinkebyMnemonic, be sure to set it in the .env file
+//const rinkebyMnemonic = process.env["RINKEBY_MNEMONIC"];
+
 //uncomment to use mainnetMnemonic, be sure to set it in the .env file
 //const mainnetMnemonic = process.env["MAINNET_MNEMONIC"]
```

Comment:
As is, I think this config will error if this stays commented out since this variable is used on line 88. Either way, the dev needs to do some set up so it's probably not a big deal. Following the README though, the dev would have created a .env file that lists all of these, so maybe we uncomment them?

Reply:
Yeah I think its ok to uncomment them!

Reply:
Done